libs/wallet/src/order-hooks/order-event-query.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from '@apollo/client';
+
+export const ORDER_EVENT_SUB = gql`
```

Comment:
nitpick (non-blocking): I would prefer this to be in the useOrderCancel file, as it is not shared.

Reply:
I would use this in useOrderEdit and maybe useOrderSubmit. eventually I would have all the order hooks living here

Reply:
We should not share queries, the idea behind GQL is to request what you need only, not everything. If we need to wait for an order and useEdit needs to check the price and useCancel does not these should be 2 separate queries. Sharing them gives us more headaches than it solves problems. Otherwise we are throwing away the majority of the value proposition of GQL. If the orders subs happen to for some reason need exactly the same field I guess we could consider sharing them but the default should always be separate queries unless you have a really good reason that they should be shared, e.g. trade list and charts do a whole bunch of data processing, which needs to not be duplicated for performance reasons, which is why those are shared.

Also chore (blocking): there are fields in this query which are unused can you please remove them. e.g. eventId, block.